src/test/resources/application.yml

```
@@ -0,0 +1,21 @@
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: create-drop # 테스트 종료 후 데이터베이스 정리
+    database-platform: org.hibernate.dialect.H2Dialect
+    show-sql: true # 테스트 시 SQL 쿼리 확인
+
+  datasource:
+    url: jdbc:h2:mem:testdb # 테스트용 인메모리 DB
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)